xbmc/pictures/GUIWindowSlideShow.cpp

```diff
@@ -737,7 +737,7 @@ bool CGUIWindowSlideShow::OnAction(const CAction &action)
 {
   switch (action.GetID())
   {
-  case ACTION_SHOW_CODEC:
+  case ACTION_PLAYER_PROCESS_INFO:
     {
       CGUIDialogPictureInfo *pictureInfo = (CGUIDialogPictureInfo *)g_windowManager.GetWindow(WINDOW_DIALOG_PICTURE_INFO);
```

Comment:
right but this has ever been wrong. for now it is important that the info will show on "o".